.github/workflows/build.yml

```
@@ -0,0 +1,139 @@
+          MATRIX_PACKAGE: ${{ matrix.package }}
+          ARCH: ${{ matrix.arch.id }}
+      - name: Push
+        run: |
+          echo "$SIGNING_KEY" | nix store sign --key-file /dev/stdin "$(< out.paths)"
+          nix copy --to "s3://wire-cache?write-nar-listing=1&ls-compression=br&log-compression=br&endpoint=s3.us-east-005.backblazeb2.com" "$(< out.paths)" -v
+        env:
+          SIGNING_KEY: ${{ secrets.SIGNING_KEY }}
+          AWS_ACCESS_KEY_ID: ${{ secrets.CACHE_AWS_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.CACHE_AWS_KEY }}
+  tests:
+    runs-on: ${{ matrix.arch.runner }}
```

actionlint (1.7.11)

[error] property "arch" is not defined in object type {test: any}

(expression)